code/datums/outfits/jobs/engineering.dm

```diff
@@ -1,9 +1,9 @@
 /decl/hierarchy/outfit/job/engineering
 	hierarchy_type = /decl/hierarchy/outfit/job/engineering
-	belt = /obj/item/storage/belt/utility/full
+	belt = /obj/item/storage/belt/utility/technomancer
 	l_ear = /obj/item/device/radio/headset/headset_eng
 	shoes = /obj/item/clothing/shoes/workboots
-	gloves = /obj/item/clothing/gloves/thick
+	gloves = /obj/item/clothing/gloves/insulated
```

Comment:
```suggestion
	gloves = /obj/item/clothing/gloves/thick
```

Lose the insuls since the insulated armor protects the body from shocks and the electrical closet already has some. Plus, they look kinda ugly with the armor.